### PR TITLE
[README] Add macOS command

### PR DESCRIPTION
**Pull request by juanjux**
Signed-off-by: Juanjo Alvarez <juanjo@sourced.tech>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,13 @@ The recommended way to run *bblfshd* is using *docker*:
 docker run -d --name bblfshd --privileged -p 9432:9432 -v /var/lib/bblfshd:/var/lib/bblfshd bblfsh/bblfshd
 ```
 
+On macOS, use this command instead to use a docker volume:
+
+```sh
+docker run -d --name bblfshd --privileged -p 9432:9432 -v bblfsh-cache:/var/lib/bblfshd bblfsh/bblfshd
```
Comment (mcuadros) [inline README.md]: cache? why is called like this?

Comment (juanjux) [reply README.md]: Well is a cache of driver images by I don't care much about the name, what do you propose?

Comment (mcuadros) [reply README.md]: Is not a cache, is the storage, of the containers and drivers.

Comment (juanjux) [reply README.md]: Oooook, PR updated.

**Pull request merged**